server/api/call.go

```diff
@@ -25,12 +25,12 @@ const (
 )
 
 type CallResponse struct {
-	Type CallResponseType
+	Type CallResponseType `json:"type,omitempty"`
 
-	Markdown md.MD       `json:"markdown,omitempty"`
-	Data     interface{} `json:"data,omitempty"`
+	Markdown md.MD                  `json:"markdown,omitempty"`
+	Data     map[string]interface{} `json:"data,omitempty"`
```

Comment:
I personally prefer a map[string]interface, since it is easier to handle. I understand that for this PR, i could have added the Post or the Error list directly as the Data, but I still feel more comfortable using map[string]interface{}. Nevertheless, I am open to revert back that change.

Reply:
IMO, `Data` is to return any data the app wants to return from a successful invocation, usually some sort of a JSON result. The pointer here should be as generic as possible, thus interface{}.

For other use-cases (returning the Form, for example) I am inclined to add typed, dedicated fields. Anyway, we can decide on this later. Do a separate cleanup when we have more use-cases implemented.